File: output/LLaMa/avatar/Java/Python/codeforces_99_A.py
import sys

input_str = input()
str_lst = list(input_str)
dotIndex = 0

for i in range(len(str_lst)):
    if str_lst[i] == '.':
        dotIndex = i
        break

if str_lst[dotIndex - 1] == '9':
    print("GOTO Vasilisa.")
else:
    next_str = str_lst[dotIndex + 1]
    if int(next_str) > 4:
        _str = input_str[:dotIndex]
        print(int(_str) + 1)
    else:
        for i in range(dotIndex):
            print(str_lst[i], end='')
        print()

File: output/LLaMa/avatar/Python/Java/codeforces_99_A.java
import java.util.Scanner;

public class codeforces_99_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        String[] parts = input.split("\\.");
        String s = String.valueOf(parts[0]);
        String p = String.valueOf(parts[1]);

        int i = Character.getNumericValue(p.charAt(0));
        if (s.charAt(s.length() - 1) == '9') {
            System.out.println("GOTO Vasilisa.");
        } else if (s.charAt(s.length() - 1) != '9' && i < 5) {
            System.out.println(s);
        } else {
            int num = Integer.parseInt(s) + 1;
            System.out.println(num);
        }
    }
}